funnel/views/project.py

```diff
@@ -74,6 +75,8 @@ class FunnelProfileProjectView(ProfileProjectView):
 
 @route('/<profile>/<project>/')
 class ProjectView(ProjectViewMixin, UrlForView, ModelView):
+    __decorators__ = [legacy_redirect]
+
```

Comment:
You need this on all classviews that handle a profile, not just ProjectView.

Reply:
Yeah, i'll do that. Was just testing with this one.

Reply:
Added it to all modelviews.